lib/jxl/fast_dct128-inl.h

```diff
@@ -12,7 +12,7 @@ constexpr size_t FastIDCTIntegerBits(FastDCTTag<128>) { return 2; }
 
 void FastIDCT(FastDCTTag<128>, const int16_t* in, size_t in_stride,
               int16_t* out, size_t out_stride, size_t count) {
-  JXL_ASSERT(count % 8 == 0);
+  JXL_DASSERT(count % 8 == 0);
```

Comment:
we might as well delete these files, btw...

Reply:
Will prepare separate PR for that.